legend-engine-xt-protobuf-pure/src/main/resources/core_external_format_protobuf/transformation/fromPure/tests/pureToProtobufTests.pure

```diff
@@ -126,18 +126,18 @@ function meta::external::format::protobuf::transformation::fromPure::tests::tran
                 '  meta.external.format.protobuf.transformation.fromPure.tests.account.Account account = 5;\n'+
                 '}'
                 , $resultForTrade.second);
-  let resultForAccount = $results->filter( r | $r.first == 'meta_external_format_protobuf_transformation_fromPure_tests_account.proto');              
+  let resultForAccount = $results->filter( r | $r.first == 'meta_external_format_protobuf_transformation_fromPure_tests_account.proto');
   assertEquals('syntax = "proto3";\n' +
                 'package meta.external.format.protobuf.transformation.fromPure.tests.account;\n' +
                 '\n' +
-                'import "google/protobuf/timestamp.proto";\n' +
-                'import "meta_external_format_protobuf_transformation_fromPure_tests_trade.proto";\n' +
+                'import "google/type/date.proto";\n' +
```

Comment:
Can we add coverage for all the defined types (Date/DateTime/StrictTime)?  With the proposed changes, seems we only cover StrictDate.

Also, it seems we don't execute pure test cases for this module (given the assertion changes)  Is this a miss on the module setup or a explicit choice?